docs/src/components/DocPage/DocPage.tsx

```diff
@@ -10,7 +10,7 @@ interface DocPageProps {
 
 const DocPage = ({ title, description, children }: DocPageProps) => (
   <DocumentTitle title={`Stardust - ${title}`}>
-    <div style={{ padding: '2rem', fontSize: '1.15rem', maxWidth: '80ch' }}>
+    <div style={{ padding: '2rem', fontSize: '1.15rem', maxWidth: '100ch' }}>
```

Comment:
Code width in prettier config in `CodeSnippet` is set to 100 chars. This makes our doc pages compatible with that.